vnc/vnc.c

```diff
@@ -2133,6 +2133,7 @@ lib_mod_set_param(struct vnc *v, const char *name, const char *value)
             (const struct xrdp_client_info *) value;
 
         g_free(v->client_layout.s);
+        v->client_layout.count = 0;
```

Comment:
Should we also set v->client_layout.s to NULL here?

Reply:
Quite right, although I'm probably going to make v->client_layout.s static anyway as it simplifies the logic quite a bit at the cost of a bit more memory.

Reply:
Memory is cheap, bugs are not :)